src/ClassDesignChallengeConstructors/Student.java

```diff
@@ -0,0 +1,15 @@
+package ClassDesignChallengeConstructors;
+
+public class Student {
+    private String firstName;
```

Comment:
Please leave an empty first row inside each class, as you did with Professor and Course. It is not an actual convention, but we either do this in each class, or we don't leave an empty row. We should have a uniform code formatting style.